components/appbar/separators.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Separators
+page_title: AppBar - Separators
+description: Separate items in the Telerik AppBar for Blazor
+slug: appbar-separators
+tags: telerik,blazor,appbar,navbar,separator,spacer
+published: True
+position: 5
+---
+
+# Separators
+
+You can visually separate the items in the Telerik AppBar for Blazor. Depending on the needs of your application you can use of the following, or a combination of them.
+
+## AppBar Separator
+
+To separate two items with a solid line, add the `<AppBarSeparator>` tag between them.
+
+### AppBar Separator Parameters
+
+The nested `AppBarSeparator` tag exposes parameters:
+
+| Parameter | Type and Default Value | Description |
+| ----------- | ----------- | ----------- |
+| `Class` | `string` | The CSS class that will be rendered on the main wrapping element of the AppBar separator. You could use that class to cascade styles. |
+| `Visible` | `bool` <br /> `true` | Specifies if the separator will be visible in the AppBar. |
+
+## AppBar Spacer
+
+To separate two items with a solid line, add the `<AppBarSeparator>` tag between them.
+
+Use the `<AppBarSpacer>` to define empty space in the AppBar which separates the items.
+
+### AppBar Spacer Parameters
+
+The nested `AppBarSpacer` tag exposes parameters:
+
+| Parameter | Type and Default Value | Description |
+| ----------- | ----------- | ----------- |
+| `Class` | `string` | The CSS class that will be rendered on the main wrapping element of the AppBar spacer. You could use that class to cascade styles. |
+| `Size` | `string` | The width of the spacer. If you add an `AppBarSpacer` tag without Size the spacer will take all available space between the [Sections]({%slug appbar-sections%}). |
```

Comment:
A more precise explanation would be that all spacers without a width will take up the same amount of the remaining available space.